deepcell/model_zoo/tracking.py

```diff
@@ -233,12 +233,35 @@ def call(self, inputs):
         return output
 
 
+class Unmerge(tf.keras.layers.Layer):
```

Comment:
Should this be moved to the `layers` module?

Reply:
I was wondering that too, there's also two other layers defined here, `Comparison` and `DeltaReshape`. Should we move all three into an existing `layers` file or a new one?

Reply:
We should probably move them into a new layers file. IDK what to name it, probably should be more about the temporal/graph nature rather than tracking.

Each of them should have tests as well, but they cannot use the standard testing routine since they have multiple inputs.

I think the `FilterDetections` layer has some good tests that are not using the `layer_test` method.

Reply:
We have  `temporal`set up already - that may be a good fit.

Reply:
I'm going to merge this PR and update the TODO checklist to include migrating and testing the layers.